week-8/Homework/mandatory/3-dog-photo-gallery/script8.js

```diff
@@ -0,0 +1,19 @@
+let button = document.querySelectorAll(".btn")[0]
+button.addEventListener('click', getImage)
+
+function getImage() {
+    fetch('https://dog.ceo/api/breeds/image/random')
+        .then(res => {
+            console.log(res);
+            return res.json();
+        })
+        .then(data => {
+            console.log(data);
+            let img = document.getElementById('dogImage');
```

Comment:
This works well for replacing the image :) Can you also try _adding_ the image (so that each time you press the button it has one more dog), rather than replacing?

Reply:
Done